config/environments/production.rb

```diff
@@ -3,6 +3,7 @@
   config.cache_store = :redis_cache_store, { url: IdentityConfig.store.redis_url }
   config.eager_load = true
   config.consider_all_requests_local = false
+  config.action_controller.asset_host = IdentityConfig.store.asset_host
```

Comment:
How does this interact with:

https://github.com/18F/identity-idp/blob/main/config/environments/production.rb#L9-L13